src/Render/RetrofitHtmlResponseAttachmentsProcessor.php

```diff
@@ -31,7 +31,7 @@ public function processAttachments(AttachmentsInterface $response)
         }
         if (isset($attachments['js']) && is_array($attachments['js'])) {
             foreach ($attachments['js'] as $key => $item) {
-                if (isset($item['type'], $item['data']) && $item['type'] === 'setting') {
+                if (is_array($item) && isset($item['type'], $item['data']) && $item['type'] === 'setting') {
```

Comment:
What could the value be if it's not an array?

Reply:
That's just to fix a phpstan message.